MapboxVision/src/test/java/com/mapbox/vision/dsl/VisionTestDsl.kt

```diff
@@ -92,4 +97,27 @@ class WhenContext {
     fun Then(displayName: String, block: () -> Unit) {
         TestCase.addCheckBlock(displayName, block = block)
     }
+
+    fun Verify(
```

Comment:
please write java/koltin-docs or comments for us

Reply:
done